<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SetupManager</class>
 <widget class="QWidget" name="SetupManager">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>651</width>
    <height>444</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QStackedWidget" name="stackedSetup">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="StartPage">
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="font">
          <font>
           <pointsize>18</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Welcome to Aklatan!</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="Line" name="line">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>This tool will help you setup Aklatan to your needs.</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Click &quot;Next&quot; to continue.</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>282</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="5" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QPushButton" name="buttonCancel0">
           <property name="text">
            <string>Cancel</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="buttonNext0">
           <property name="text">
            <string>Next</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="DatabaseSetup">
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="1" column="0">
        <widget class="Line" name="line_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label_4">
         <property name="font">
          <font>
           <pointsize>18</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Database Setup</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QStackedWidget" name="stackedDB">
         <property name="currentIndex">
          <number>1</number>
         </property>
         <widget class="QWidget" name="dbPage1">
          <layout class="QGridLayout" name="gridLayout_4">
           <item row="1" column="0">
            <spacer name="verticalSpacer_2">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="5" column="0" colspan="3">
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <widget class="QPushButton" name="buttonCanceldb1">
               <property name="text">
                <string>Cancel</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer_2">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QPushButton" name="buttonBackdb1">
               <property name="text">
                <string>Back</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="buttonNextdb1">
               <property name="text">
                <string>Next</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item row="3" column="0">
            <widget class="QLineEdit" name="linedbName"/>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="label_6">
             <property name="font">
              <font>
               <pointsize>10</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Name of the database Aklatan is going to use:</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item row="4" column="0">
            <spacer name="verticalSpacer_3">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string>Aklatan is going to use a database to store everything in the library. This includes books, users, and the information about them.</string>
             </property>
             <property name="scaledContents">
              <bool>false</bool>
             </property>
             <property name="wordWrap">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QLabel" name="label_7">
             <property name="text">
              <string>.db</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="dbPage2">
          <layout class="QGridLayout" name="gridLayout_5">
           <item row="12" column="0">
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <item>
              <widget class="QPushButton" name="buttonCanceldb2">
               <property name="text">
                <string>Cancel</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer_3">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QPushButton" name="buttonBackdb2">
               <property name="text">
                <string>Back</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="buttonNextdb2">
               <property name="text">
                <string>Next</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="label_9">
             <property name="text">
              <string>Username for the admin account:</string>
             </property>
            </widget>
           </item>
           <item row="10" column="0">
            <widget class="QLabel" name="labelCheck">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item row="11" column="0">
            <spacer name="verticalSpacer_5">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="5" column="0">
            <widget class="QLabel" name="label_10">
             <property name="text">
              <string>Password for the admin account:</string>
             </property>
            </widget>
           </item>
           <item row="7" column="0">
            <widget class="QLabel" name="label_11">
             <property name="text">
              <string>Verify admin password:</string>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QLineEdit" name="lineUsername">
             <property name="echoMode">
              <enum>QLineEdit::Normal</enum>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <spacer name="verticalSpacer_4">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="label_8">
             <property name="font">
              <font>
               <pointsize>10</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Set a username and password for the library admin account. This account will have full access to the library interface.</string>
             </property>
             <property name="wordWrap">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="9" column="0">
            <widget class="QLineEdit" name="linePassCheck">
             <property name="echoMode">
              <enum>QLineEdit::Password</enum>
             </property>
            </widget>
           </item>
           <item row="6" column="0">
            <widget class="QLineEdit" name="linePassword">
             <property name="echoMode">
              <enum>QLineEdit::Password</enum>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="SettingSetup">
      <layout class="QGridLayout" name="gridLayout_6">
       <item row="7" column="0">
        <widget class="QLineEdit" name="lineLate"/>
       </item>
       <item row="1" column="0">
        <widget class="Line" name="line_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_13">
         <property name="text">
          <string>Setup the penalties that Aklatan would automatically account for at book return.</string>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <spacer name="verticalSpacer_7">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="8" column="0">
        <widget class="QLabel" name="label_15">
         <property name="text">
          <string>Penalties for a book being returned damaged:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label_12">
         <property name="font">
          <font>
           <pointsize>18</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Settings Setup</string>
         </property>
        </widget>
       </item>
       <item row="11" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QPushButton" name="buttonCancelSettings">
           <property name="text">
            <string>Cancel</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_4">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="buttonBackSettings">
           <property name="text">
            <string>Back</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="buttonNextSettings">
           <property name="text">
            <string>Next</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="9" column="0">
        <widget class="QLineEdit" name="lineDamaged"/>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="label_14">
         <property name="text">
          <string>Penalties for a book being returned late:</string>
         </property>
        </widget>
       </item>
       <item row="10" column="0">
        <spacer name="verticalSpacer_6">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_16">
         <property name="text">
          <string>How many days should a book be considered late:</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLineEdit" name="lineDue"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page">
      <layout class="QGridLayout" name="gridLayout_7">
       <item row="4" column="1">
        <widget class="QPushButton" name="buttonFinish">
         <property name="text">
          <string>Finish</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label_17">
         <property name="font">
          <font>
           <pointsize>18</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Setup Complete</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_18">
         <property name="text">
          <string>Aklatan will now start. Please login with the admin account you created.</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <spacer name="verticalSpacer_8">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="4" column="0">
        <spacer name="horizontalSpacer_5">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="0" colspan="2">
        <widget class="Line" name="line_4">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
